commands/user.bee.inc

```diff
@@ -98,6 +187,180 @@ function users_bee_callback($arguments, $options) {
   );
 }
 
+/**
+ * Command callback: Create a user.
+ */
+function user_bee_create_callback($arguments, $options) {
+  $user = user_load_by_name($arguments['username']);
+  if ($user) {
+    bee_message(bt("User account '!user' allready exits.", array(
+      '!user' => $arguments['username'],
+    )), 'error');
+    return;
+  }
+  if(empty($arguments['mail'])){
+    bee_message(bt("Mail argument missing!", array()), 'error');
+    return;
+  }
+  $check = db_select('users', 'u')
+    ->fields('u', array('uid'))
+    ->condition('mail', $arguments['mail'], '=')
+    ->execute()
+    ->fetchField();
+
+  if($check != '') {
```

Comment:
You could use [user_load_by_mail()](https://docs.backdropcms.org/api/backdrop/core%21modules%21user%21user.module/function/user_load_by_mail/1) here